pr-scripts/create_pr.sh

```diff
@@ -42,6 +42,10 @@ if [[ $JOB_NAME =~ "prow-deck-tooling" ]]; then
     CHANGED_FILE="Prow controlplane Helm chart"
     CHANGED_COMPONENT="Prow component images"
 fi
+if [[ $JOB_NAME =~ "quarterly-minimal-image-rebuild" ]]; then
```

Comment:
set job name as a var and re-use in multiple places so we can change it once if we need to?